modules/foliate/hm.nix

```
@@ -0,0 +1,34 @@
+{ config, lib, ... }:
+
+{
+  options.stylix.targets.foliate.enable =
+    config.lib.stylix.mkEnableTarget "Foliate" true;
+
+  config =
+    lib.mkIf (config.stylix.enable && config.stylix.targets.foliate.enable)
+      {
+
+        # Generate the theme
+        xdg.configFile."com.github.johnfactotum.Foliate/themes/stylix.json".text =
+          with config.lib.stylix.colors;
+
+          builtins.toJSON {
+            label = "Stylix";
+            light = {
+              fg = "#${base00}";
+              bg = "#${base05}";
+              link = "#${base0D}";
+            };
+            dark = {
+              fg = "#${base05}";
+              bg = "#${base00}";
+              link = "#${base0D}";
+            };
+          };
+
+        # Select the theme
```

The code here is self explanatory, the comments are unnecessary.
```suggestion
        xdg.configFile."com.github.johnfactotum.Foliate/themes/stylix.json".text =
          with config.lib.stylix.colors;

          builtins.toJSON {
            label = "Stylix";
            light = {
              fg = "#${base00}";
              bg = "#${base05}";
              link = "#${base0D}";
            };
            dark = {
              fg = "#${base05}";
              bg = "#${base00}";
              link = "#${base0D}";
            };
          };
          
```